server/shadow/shadow_client.c

```diff
@@ -248,6 +248,14 @@ BOOL shadow_client_post_connect(freerdp_peer* peer)
 		}
 	}
 
+	if (subsystem->ClientConnect)
+	{
+		if (subsystem->ClientConnect(subsystem, peer))
```

Comment:
Yeah It should be. 
I want this callback as the last line in post connect, if subsystem->ClientConnect exists.